sklearn/neighbors/tests/test_lof.py

```diff
@@ -30,9 +30,12 @@
 iris.target = iris.target[perm]
 
 
-def test_lof():
+def test_lof(global_dtype):
     # Toy sample (the last two samples are outliers):
-    X = [[-2, -1], [-1, -1], [-1, -2], [1, 1], [1, 2], [2, 1], [5, 3], [-4, 2]]
+    X = np.asarray(
+        [[-2, -1], [-1, -1], [-1, -2], [1, 1], [1, 2], [2, 1], [5, 3], [-4, 2]],
+        dtype=global_dtype,
+    )
```

Comment:
it changes the original test. before it was an array of int. Now it's either float32 or float64

Reply:
Following up on this: we still want to convert, right?

Reply:
Yes I guess